source/GlassView.Export.Test/ExportTest.cs

```diff
@@ -0,0 +1,13 @@
+using BenchmarkDotNet.Reports;
+
+namespace GlassView.Export.Test;
+
+public class ExportTest
+{
+    private const String report = @"Quantities.Benchmark.DeserializationBenchmark-report.json";
```

Comment:
There's no way to deserialize this genuine Benchmark.Net exported json file back to a summary.
Hence I can't write meaningful tests for the exporter...

Reply:
For me it's enough if we manage to import them. After all, everybody will still have access to their original benchmarks and our output. So there should be no problems.

Therefore, I would unit test the import and leave the export aside.